lib/js/src/protocol/SdlProtocolListener.js

```diff
@@ -165,14 +165,15 @@ class SdlProtocolListener {
 
 
     /**
+     * Sets the getDesiredVideoParams getter
      * @param {function} getter
      */
     setGetDesiredVideoParams (getter) {
         this._getDesiredVideoParams = getter;
     }
 
     /**
-     * Invoke the getSessionId getter
+     * Invoke the getDesiredVideoParams getter
      * @returns {Number} sessionId
```

Comment:
Is this the correct `@returns` documentation?

Reply:
```@returns {VideoStreamingParameters} ```